src/pybind/mgr/dashboard/settings.py

```diff
@@ -102,6 +102,17 @@ def py2ceph(pytype):
     return cmd_list
 
 
+def options_schema_list():
```

Comment:
By getting the options from the `_OPTIONS_COMMAND_MAP` you will generate options with the form `dashboard get-OPTION_NAME`.

I think the correct implementation should be:
```python
def options_schema_list():
    def filter_attr(member):
        return not inspect.isroutine(member)

    result = []
    for option, value in inspect.getmembers(Options, filter_attr):
        if option.startswith('_'):
            continue
        result.append({'name': option, 'default': value[0]})

    return result
```

Reply:
Thanks, updated